realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -334,6 +331,44 @@ public void changePassword(String newPassword) throws ObjectServerError {
     }
 
     /**
+     * Changes another user's password. This is done synchronously and involves the network, so calling this method on the
+     * Android UI thread will always crash.
+     * <p>
+     * This user needs admin privilege in order to change someone else's password.
+     *
+     * <b>WARNING:</b> Changing a users password using an authentication server that doesn't use HTTPS is a major
```

Comment:
`users` -> `user's` `using` -> `through`
hanging a user`s password through an authentication server that doesn't use HTTPS is a major